gmmp/settings.py

```diff
@@ -204,10 +204,11 @@
     "QN": {'name': _('Belgium - Flemish'), 'alpha3': 'VLG'},
     "UK": None,
     "QO": {'name': _('England'), 'alpha3': 'ENG'},
-    "QP": {'name': _("North Ireland"), 'alpha3': 'NIR'},
+    "QP": {'name': _("Northern Ireland"), 'alpha3': 'NIR'},
```

Comment:
This change won't have any impact on existing region mappings i.e. https://github.com/CodeForAfrica/gmmp/blob/a540adcbef9d1b533883a0734ff9c278adb9a1e7/forms/migrations/0043_update_country_region.py#L7?

Reply:
I don't think so. I think it's just used for the display.

Reply:
Cool, lets just test it (switch your country locally to Northern Ireland and add a story) @esirK .

Reply:
This works.